joplin/js/editor.js

```diff
@@ -262,29 +255,113 @@ $(function() {
     }
   }
 
-  var enButton = $('#en');
-  enButton.click(function() {
-    changeLanguage('en');
+  if (localStorage.selected_lang) {
+    state.currentLang = localStorage.selected_lang;
```

Comment:
The relationship between these is confusing to me, I know we've got a lot going on that uses them so I don't really want to touch it, but it could be nice to make an issue to investigate/clean it up sometime.